design-reviews/recipes/engagement-process.md

```diff
@@ -0,0 +1,129 @@
+# Incorporating Design Reviews into an Engagement
+
+## Introduction
+
+Design reviews should not feel like a burden.  Design reviews can be easily incorporated into your dev crew process with minimal overhead.  
+
+- Only create design reviews when needed.  Not every story or task requires a full blown design review
+- Leverage this guidance and make changes to fit in best with your team.  Every team works a bit different and has different tweaks that work best for them.
+- Leverage SME's as needed during design reviews. Not every story needs SME or leadership sign-off.  Most design reviews can be fully executed within your own dev crew.
+
+The following is are some guidelines on how you and your team can start incorporating design reviews into your normal day to day agile process.
+
+## Table of Contents
+
+- [Envisioning / ADS](#envisioning-/-ads)
+- [Game Plan](#game-plan)
+- [Sprint Planning](#sprint-planning)
+- [Sprint Backlog Grooming](#sprint-backlog-grooming)
+- [Sprint Retrospectives](#sprint-retrospectives)
+- [Wrap-up Sprints](#wrap-up-sprints)
+
+## Envisioning / ADS
+
+Early on within an engagement process we are working with customers to understand their unique goals and objectives.  We are defining requirements to understand definition of done.  We are diving deep into their existing infrastructure and architecture to understand constraints.
+
+During this time we are uncovering as many unknowns as possible to help us jointly generate a design to meet the customers goals.  We are leveraging all the new found information an incorporating it into our game plan design for the customer engagement.
+
+> **Tip**: We have not uncovered all the unknowns at this point. You will continue to learn and discover more during the lifetime of your engagement
+
+## Game Plan
+
+The game plan document is our guiding star throughout the engagement.  The game plan document should include the following sections and answer the following questions.
+
+### Deliverable

```

Comment:
Does this section map to the Game Plan template v4 or purposely higher level?

Reply:
@fnocera - any guidance here?

Reply:
I attempted to distill it down to the highlights vs going too deep into the game plan specifics.